### DISCUSSION TITLE
Is there a bootstrap example that illustrates how to create documentation the same way getbootstrap does

### DISCUSSION THREAD

**Discussion by plastikfan** [twbs/bootstrap#34987] [Q&A] [answered]
I am using Jekyll and Bootstrap to create some documentation. It occured to me that the layout of getbootstrap is just perfect for what I need to create and I have looked at [examples](https://getbootstrap.com/docs/5.1/examples) to see if there was an example of this format, ie a 3 column layout, with a verticle menu on the left, a page table of contents on the right, contents in the middle and a fat footer.

I am taking this approach as I want to write documentation __and__ learn bootstrap in the process, that is why I am not using something like docusaurus or gitbook which does everything for you and you just right the copy in markdown.

And, a question to bootstrap, what documentation platform do you use if any or it is all just straight html/css and of course bootstrap itself. I am just wondering if me chasing down jekyll is the right approach given all I want to do is write documentation and learn bootstrap. Having to learn jekyll, liquid and front matter is just a necessary evil and if I could this without them, I would, thanks.

**Comment by coliff** [comment, marked as answer]
hey there. I'd personally recommend checking out [Hugo ](https://gohugo.io/)- a static-site generator similar to Jekyll and it's used to build the Bootstrap site/docs. If you clone/download the Bootstrap repo then you can build the site/docs locally and see how it was all made and feel free to copy and adapt parts to your site.

Jekyll is still a good static site generator too, but I'd recommend Hugo over it.

**Reply by plastikfan** [reply]
Hi @coliff, you are a hero, thank you. I have just started getting in to the weeds of Jekyll but not too far to back out and change course. I didnt think of just cloning bootstrap site, seems so obvious now that you have just mentioned it. Does Hugo have a templating engine a la liquid/front matter. It would be great if you could use front matter/liquid with hugo, as I have spent a bit of time learning that already, but if not then so be it. I'm gonna take a look at Hugo now and see what it offers.

**Reply by coliff** [reply]
Yes, if you know a bit of Jekyll templating already then you'll have no problem at all adapting to Hugo - it's very similar - both use front-matter and liquid-style partials etc. Good luck and have fun!

**Reply by plastikfan** [reply]
Thanks @coliff, your timing is impeccable as I was just on the cusp of really diving head first into Jekyll. Your tip of reusing getbootstrap instead of try to reproduce it all myself is a real gem 😄
